MeshLib/MeshGenerators/ConvertRasterToMesh.cpp

```diff
@@ -142,14 +147,24 @@ MeshLib::Mesh* ConvertRasterToMesh::constructMesh(const double* pix_vals, const
 					quad_nodes[1] = nodes[node_idx_map[index + 1]];
 					quad_nodes[2] = nodes[node_idx_map[index + width + 1]];
 					quad_nodes[3] = nodes[node_idx_map[index + width]];
-					elements.push_back(new MeshLib::Quad(quad_nodes, mat));
+					elements.push_back(new MeshLib::Quad(quad_nodes));
+					mat_ids.push_back(mat);
 				}
 			}
 		}
 	}
 	delete [] node_idx_map;
 
-	return new MeshLib::Mesh("RasterDataMesh", nodes, elements); // the name is only a temp-name, the name given in the dialog is set later
+	// the name is only a temp-name, the name given in the dialog is set later
+	MeshLib::Mesh* mesh(new MeshLib::Mesh("RasterDataMesh", nodes, elements));
+	auto opt_pv = mesh->getProperties().createNewPropertyVector<int>(
+		"MaterialIDs", MeshLib::MeshItemType::Cell);
+	if (opt_pv) {
```

Comment:
Would abort if the material ids property vector could not be created, otherwise the returned mesh will sometimes have material ids and sometimes not.